erpnext/buying/doctype/purchase_order_item/purchase_order_item.json

```diff
@@ -64,6 +64,11 @@
   "total_weight",
   "column_break_40",
   "weight_uom",
+  "cultivation_tax_details",
+  "flower_weight",
+  "leaf_weight",
+  "plant_weight",
+  "cultivation_weight_uom",
```

Comment:
There was supposed to be a column break `column_break_55` before `cultivation_weight_uom`, check the bloomstack_core file.